### ISSUE TITLE
Pedal text not rendering at all when I insert a marking with said text

### ISSUE THREAD

**Issue by DaddyLudwig** [musescore/MuseScore#20646]
### Issue type

Engraving bug (incorrect score rendering)

### Bug description

The line renders fine, the end hook (in my case the star) renders fine, but the Ped. text does not render correctly, I can't see it. It happens every time I click on the Pedal (with ped and asterisk) item on the palette.

### Steps to reproduce

1. Select some notes
2. Add the Pedal (with ped and asterisk) item from the palate to the selection
3. See that the text doesn't render correctly, but the rest of the marking does

### Screenshots/Screen recordings

https://github.com/musescore/MuseScore/assets/6530761/c28c4c1e-79fa-48e2-a0db-a1a171efe266

### MuseScore Version

MuseScore 4.2

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Windows 10

**Comment by jeetee** [comment]
Workaround: after adding the line to the score, manually put `<sym>keyboardPedalPed</sym>` in the line begin text field.

Side note: the bug has to do with how the palette element gets applied, because Ctrl/Cmd+Shift+dragging the manually corrected element from the score back into the palettes and then applying it somewhere else shows the same missing start text.

**Comment by jeetee** [comment]
Additional symptom:
When lengthening the pedal line over a system using Shift+right or drag; the first part of the pedal line (visually) disappears. 

This also happens when applying the pedal (from keyboard palette) over a preselected range that crosses a system. But (strangely enough) does not happen when using the pedal element from the "lines" palette.

The pedal line from the lines palette shows up completely correct if applied immediately over a range; if not (apply to single element), it is broken the same way as described above.
For the pedal line from the keyboard palette however, there seems to be no difference whether initially applied to a range or a single element.

**Comment by MarcSabatella** [comment]
I saw a similarly strange result the other day and was attempting to put together steps to reproduce the problem when suddenly it fixed itself.

In my case, it started when I tried creating a custom pedal marking by starting from the "Ped___*" version then removing the end text (leaving just "Ped___*), then adding this back to the palette.  This custom version then would not show the Ped text.  But then after resetting the palette and restarting to try to reproduce the issue, it went away, I cannot reproduce anymore.

So I'm not sure exactly what is going on, but my guess is that it is related to https://github.com/musescore/MuseScore/pull/19666
